### ISSUE TITLE
PSR Standard for annotations

### ISSUE THREAD

**Issue by DavidGarciaCat** [JPGSP/blog#5]
Although is not a blocker, there is a good practice for all PHP annotations: to add an empty like between each "block" of annotations, so an annotation like this:

https://github.com/JPGSP/blog/blob/master/src/BlogBundle/Entity/Comment.php#L45-L46

```php
    /**
     * @param string $string
     * @return self
     */
```

should be like this:

```php
    /**
     * @param string $string
     *
     * @return self
     */
```

Please note the same kind of annotations should be grouped together, like this:

```php
    /**
     * @param string $string
     * @param int    $integer
     *
     * @return self
     */
```

**Comment by JPGSP** [comment]
Issue fixed in the last commit (reference e9ebd71371bcdc5b38caef6eb99abf48a300d8c5)

**Comment by DavidGarciaCat** [comment]
#9 

PHP Annotations fixes are not related with ORM annotations 😄

Please try to be precise and concise with your commits, is much better to add 20 commits with small changes (that we can revert easily) than a single commit (that means we might need to revert the whole update)
